envfile/tasks/main.yaml

```diff
@@ -2,3 +2,4 @@
   template:
     src: envfile.j2
     dest: '{{ path }}'
+    mode: '0600'
```

Comment:
This is technically a BC break, but since env files often contains secrets it cas also be considered as a security fix.